v19.1/install-cockroachdb-linux.html

```diff
@@ -130,7 +130,7 @@ <h2>Build from Source</h2>
         </tr>
         <tr>
           <td>Go</td>
-          <td>Version 1.11.4 or higher is required.</td>
```

Comment:
I just bumped it again to 1.11.6.

Reply:
For next week's rc, you mean? If so, I'll do this again :)